src/js/components/sorter/Sorter.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { ButtonToolbar, ToggleButtonGroup, ToggleButton } from 'react-bootstrap';
+import './sorter.style.less';
+
+class Sorter extends Component {
+
+  state = {
+    sortby: this.props.sortby
+  };
+
+  clickHandler = () => {
+    if (event.target.value) {
+      this.setState({sortby: event.target.value});
+      this.props.changeHandler(this.state);
+    }
+  }
+
+  render () {
+    return (
+      <div className="sorter" onClick={this.clickHandler}>
```

Comment:
onClick on div will not return a value from selected option, as you're using https://react-bootstrap.github.io/components/button-group/ there's in a doc that:

```      <ToggleButtonGroup
        type="checkbox"
        value={this.state.value}
        onChange={this.handleChange}
      >```

`ToggleButtonGroup` os place where you should put this handler